service/src/main/java/org/ehrbase/dao/access/interfaces/I_FolderAccess.java

```diff
@@ -94,6 +95,14 @@ static I_FolderAccess retrieveInstanceForExistingFolder(I_DomainAccess domainAcc
         return FolderHistoryAccess.retrieveInstanceForExistingFolder(domainAccess, folderId, timestamp);
     }
 
+    /**
+     * Creates a new directory object with a given structure and returns a valid Object_Version_Id containing the given
+     * system identifier and version part.
+     *
+     * @return Object_Version_Id for new root directory folder
+     */
+    ObjectVersionId create();
```

Comment:
Did you define this to circumvent the fix `UUID` return type for `commit(..)` set by the `I_SimpleCRUD` interface, right? Clever :) I was just trying to use `ObjectVersionId` in the EHR access layer and had the same problem.
In the end I'm wondering if it would make more sense to just add a general set of CRUD methods which are returning `ObjectVersionId` to the `I_SimpleCRUD` interface (or to add a separate interface class like `I_VersionedCRUD`). So it can be used consistently between all access classes (of versioned objects). But I think if it works, it is fine and okay for this case here.